app/lib/presenters/complaint.rb

```diff
@@ -0,0 +1,49 @@
+module Presenters
+  class Complaint
+    def initialize(form_builder_payload:)
+      @data = form_builder_payload[:submissionAnswers]
+    end
+
+    def optics_payload
+      {
+        RequestDate: @data.fetch(:submissionDate, Date.today),
+        Customer: customer_data,
+        Details: @data.fetch(:complaint_details, ''),
+        Location: @data.fetch(:complaint_location, '')
+      }.merge(constant_data)
+    end
+
+    private
+
+    def customer_data
+      {
+        FullName: @data.fetch(:full_name, ''),
```

Comment:
This field is not documented in optics 
https://icasework.atlassian.net/wiki/spaces/UsefulFeedback/pages/15171615/CreateCase+API

There looks to be `Customer.FirstName` and `Customer.Surname`

Reply:
Its mentioned in other documentation so we'll have to check it in real life